app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/SkillListingPresenter.kt

```diff
@@ -98,6 +98,12 @@ class SkillListingPresenter(val skillListingFragment: SkillListingFragment) : IS
             if (metricsData != null) {
                 metrics.metricsList.clear()
                 metrics.metricsGroupTitles.clear()
+                if (metricsData?.staffPicks != null && metricsData?.staffPicks?.size as Int > 0) {
```

Comment:
Remove `as Int`

Reply:
@iamareebjamal As this is a nullable receiver, Android Studio shows a warning that this call corresponds to a dot qualified call, which is not allowed on a nullable receiver.
One way to fix this is to declare a local val, say 'size',  and then cast size to 
Int using ```if (size is Int) ``` and then checking for ```size > 0``` So, should I change this?

Reply:
`metricsData?.staffPicks?.size != null && metricsData?.staffPicks?.size > 0`

Reply:
'Smart cast to 'Int' is impossible because ```metricsData?.staffPicks?.size``` is a complex expression.'
This can be overcome by assigning this to a separate var, say size, and then applying safe cast on size and checking for the condition.

Reply:
'staffPicks' is an Array. Even if I convert it 'toTypedArray()', it will still need a cast because of a nullable receiver. The other option is to use !! after checking if ```metricsData?.staffPicks != null``` which is not safe to use.

Reply:
> The other option is to use !! after checking if metricsData?.staffPicks != null which is not safe to use.

Your statement is self-contradictory. It is definitely safe if you have checked that it is not null. Use !! but add a todo statement to make null safe